pkg/server/bootstrap/identity.go

```diff
@@ -157,26 +160,48 @@ func wildcardIdentitiesResolver(ids *identities,
 				continue
 			}
 
-			apiExport, err := kcpClient.ApisV1alpha1().APIExports().Get(ctx, name, metav1.GetOptions{})
+			apiExportIdentity, err := apiExportIdentityProviderFn(ctx, name)
 			if err != nil {
 				errs = append(errs, err)
 				continue
 			}
-			if apiExport.Status.IdentityHash == "" {
+			if apiExportIdentity == "" {
 				errs = append(errs, fmt.Errorf("APIExport %s is not ready", name))
 				continue
 			}
 
 			ids.lock.Lock()
-			ids.groupResourceIdentities[gr] = apiExport.Status.IdentityHash
+			ids.groupResourceIdentities[gr] = apiExportIdentity
 			ids.lock.Unlock()
 
-			klog.V(2).Infof("APIExport %s|%s for resource %s.%s has identity %s", logicalcluster.From(apiExport), name, gr.Resource, gr.Group, apiExport.Status.IdentityHash)
+			klog.V(2).Infof("APIExport %s|%s for resource %s.%s has identity %s", tenancyv1alpha1.RootCluster, name, gr.Resource, gr.Group, apiExportIdentity)
 		}
 		return errorsutil.NewAggregate(errs)
 	}
 }
 
+func apiExportIdentityProviderFn(kcpClient kcpclient.Interface, kubeClusterClient kubernetes.ClusterInterface) func(ctx context.Context, apiExportName string) (string, error) {
+	return func(ctx context.Context, apiExportName string) (string, error) {
+		if kubeClusterClient != nil {
+			actualApiExportIdentitiesConfigMap, err := kubeClusterClient.Cluster(configshard.SystemShardCluster).CoreV1().ConfigMaps("default").Get(ctx, configMapName, metav1.GetOptions{})
```

Comment:
What makes this the "actual" map? As opposed to a fake map?

Reply:
will rename to something else